### ISSUE TITLE
Request to move config files out of save folder

### ISSUE THREAD

**Issue by darchitectlp** [LogicTechCorp/NetherEx-Archive#243]
I just ran into plains generating in the nether (I'm playing Roguelike Adventures and Dungeons) and I guess they had an old version of Netherex in it. So I updated Netherex, wiped Dim-1 and let it load fresh. I guess that your mod copied the faulty configs into my save folder (including the plains.json) into your new config folder in the save folder. Could NOT figure out why, after wiping the netherex config AND the nether repeatedly, I kept seeing plains... Finally found a closed issue here where you mention that you've moved configs into the individual world saves folder.

Seems like a bad idea all around... Also, your warning screen that pops up makes no mention of new configs being moved outside the configs folder. What are pack makers supposed to do when they want the configs edited and distributed? They can't do that if it's in a save folder, correct? Anyway, just offering some food for thought.

And thanks for all your hard work. This is a fantastic mod!

**Comment by LogicTechCorp** [comment]
The configs used to be located in the default config folder, but they were moved to the world saves so that each world could have custom biomes. I'll take a look into adding a config option that will allow for custom configs to be copied from the default config folder to the world saves. I'll also update the description on CurseForge with more information once the feature is implemented.

**Comment by LogicTechCorp** [comment]
I've resolved this issue in the 1.14 and 1.15 versions by adding a global biome/data pack config folder. Since 1.12 is before data packs were added, I can't use the same fix. I am looking into what the best approach would be to implement something similar without causing game breaking issues.

**Comment by LogicTechCorp** [comment]
I've now resolved this in the 1.12 version by adding a global biome config folder to the NetherEx config folder. It is disabled by default and needs to be enabled in the NetherEx config. It is disabled by default as to not cause issues with preexisting worlds which use per world biome configs. I've also included a config option to disable per world biome configs. If both are enabled, per world biome configs will overwrite the global configs.
